core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooLock.java

```diff
@@ -90,21 +92,18 @@ public String toString() {
   private String createdNodeName;
   private String watchingNodeName;
 
-  public ZooLock(ZooReaderWriter zoo, String path, UUID uuid) {
-    this(new ZooCache(zoo), zoo, path, uuid);
+  public ZooLock(AccumuloConfiguration conf, String path, UUID uuid) {
+    this(conf.get(Property.INSTANCE_ZK_HOST),
+        (int) conf.getTimeInMillis(Property.INSTANCE_ZK_TIMEOUT),
+        conf.get(Property.INSTANCE_SECRET), path, uuid);
   }
 
   public ZooLock(String zookeepers, int timeInMillis, String secret, String path, UUID uuid) {
```

Comment:
The only place this constructor is called is in ZooLockIT, with the same args over and over again. It would be simpler to delete this constructor, and update ZooLockIT use a ConfigurationCopy instance of AccumuloConfiguration with the config properties already set up in a `@Before` method, so ZooLockIT can just use the other constructor.